ColorPairUnitTests.py

```diff
@@ -0,0 +1,19 @@
+import ColorPairOperations
+  
+def test_number_to_pair(pair_number,expected_major_color, expected_minor_color):
```

Comment:
Think of names for the tests, which include their expectations. Do any additional test possibilities emerge?